### PR TITLE
Rn2224n

### PR DESCRIPTION
**Pull request by khanzadyan**
Details of the product update v22.24 from 21st of June 2022.

### CODE REVIEW

**Review by luzhok — APPROVED** [review]

File: content/_releases/22/24.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+*   You can use custom `Dockerfile` to define the build process in the `Node.js` components. It must be in the root directory of your component code.
+*   If you have used a custom `Dockerfile` you might have noticed that it was not used in the build process. Now the build process **will check for the existence of it and use it**. Which means if you have not updated your `Dockerfile` recently, chances are your component build might fail or the newly built component might not function as expected. **Check your `Dockerfile` before deploying or remove it from the component code if custom build parameters are not necessary.**
+*   We now support `Node.js` version 18.
+*   **IMPORTANT**: custom `Dockerfile` is not yet supported for Java components.
+
+## Fixed bugs
+
+*   Fixed the bug in the object and attachment handling service. Now the service will retry streams on failure and will handle errors properly.
+*   Fixed the user interface bug when the required field symbol (`*`) would not show after the metadata reload in case the mandatory fields were controlled by a checkbox.
```
Comment (KMobbs) [inline content/_releases/22/24.md:57]: This needs reworking - very difficult to understand. Suggest sthg like below:

Mandatory fields in the flow designer views will be consistently displayed, even when the fields are check-boxes and even after reloading metadata.

**Review by KMobbs — APPROVED** [review]

File: content/_releases/22/24.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+To improve component deployment and build process we modernised the `appbuilder`
+and `apprunner` docker images to address potential vulnerabilities in the `Node.js`
+packages. All platform services, as well as the `Node.js` based integration components
+are built with these images.
+
+The modernisation brought some important changes which might affect you when you
+deploy the new version of your component code.
+
+*   You can use custom `Dockerfile` to define the build process in the `Node.js` components. It must be in the root directory of your component code.
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_releases/22/24.md:49]: ty

File: content/_releases/22/24.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+*   If you have used a custom `Dockerfile` you might have noticed that it was not used in the build process. Now the build process **will check for the existence of it and use it**. Which means if you have not updated your `Dockerfile` recently, chances are your component build might fail or the newly built component might not function as expected. **Check your `Dockerfile` before deploying or remove it from the component code if custom build parameters are not necessary.**
```
[the remark this answers is not in the document]
Comment (khanzadyan) [reply content/_releases/22/24.md:50]: done. Thanks

File: content/_releases/22/24.md (hunk 1 above)
Comment (khanzadyan) [reply content/_releases/22/24.md:57]: Decided to leave as it is.

**Pull request merged**